### DISCUSSION TITLE
MPD music server

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#984] [General]
So I am playing with MPD music server + its android app and I really miss the workflow and usability of tauon like playlists creation and management, queue, favorite songs, etc.

I wish there was a tauon android app that support mpd too..

**Comment by Taiko2k** [comment]
Unfortunately I don't have interest in Android app development. I find it quite unfun.

Even if I got a good Tauon for Android app, there's not really much incentive to support a separate 'ecosystem' such as MPD. Maybe it could be done, but I'm really not interested myself.

**Reply by Solarunit** [reply]
gotcha! Thanks for reply, love using tauon anyway :slightly_smiling_face:
